api/payload.go

```diff
@@ -168,3 +168,16 @@ type Signal struct {
 type ErrorResponse struct {
 	Message string `json:"msg"`
 }
+
+// LogEntry is the payload for the StreamLog data.
+type LogEntry struct {
+	// Source is the source of the log entry. One of "shim" or "runtime".
+	Source string `json:"source"`
```

Comment:
I'm struggling to understand how you'd discriminate between multiple proxy clients since there doesn't seem to be a requirement that a `LogEntry` is unique with respect to each client?

Client 1:
```
LogEntry{Source: "shim", ContainerID: "", Level "debug", Message: "foo"}
```


Client 2:
```
LogEntry{Source: "shim", ContainerID: "", Level "debug", Message: "foo"}
```